src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
According to the checklist, you should avoid creating several instances of the same class if you can use only one instance for your purposes. Here, a new `Random` object is created every time `getRandomBall()` is called, which is not necessary. Instead, you can create a single instance of `Random` as a class-level field and reuse it.